TS_HW1/tsconfig.json

```diff
@@ -11,8 +11,8 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es2016",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
-    // "lib": [],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+    "target": "ESNext",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
Compiler by default use es2016 in which these methods don't exist. Try "es2017" instead. That should help.

Reply:
Only "esnext" knows 'toSorted' method for arrays. Any others marked this method as not existed.
And compilations errors are the same.

Reply:
Did you try new tsconfig file? Is the result the same?

Reply:
Yes, I tried the config from last lesson, but two mistakes are still appearing during compilation
![image](https://github.com/olexey81/TypeScript/assets/128589522/5c9ca60a-3da8-4baa-8867-0ceeadd892cc)

Reply:
Hm, it's weird. All file is the same, right? The 'module' field is the same as well, isn't it?

Okay. What about node version? Which one do you use?

Reply:
I've tried to use config with no changes, then tried all values (ES3, ES6, .... , ESNEXT) for both "target" and "module" fields. No changes happened, 2 errors during compilation.
Node.js  is v20.10.0

Reply:
Try add es2017.object to the compilerOptions.lib array in your tsconfig.json. May be it will help. I don't know for sure, it's weird.

Reply:
![image](https://github.com/olexey81/TypeScript/assets/128589522/1a131f8a-7034-46d0-8b9a-02d6b29ef1f4)
after updating ts to 5.3.3 one error was dissapear. 
then, only with "target": "ES6", "lib": ["ESNext"] there is no any errors in ts code.
but with command tsc src\ts_hw2\hw_2.ts it's error with "toSorted" method for arrays again.
and with command tsc (for all *.ts) files - no any error messages :-) some mistics
![image](https://github.com/olexey81/TypeScript/assets/128589522/ab5ae7d4-1e3a-4887-88e2-40d194926f02)

Reply:
Hmm, it's interesting. It seems like there might be some issue with the cash.